### ISSUE TITLE
Email notification is not correct

### ISSUE THREAD

**Issue by SamuelHassine** [LuatixHQ/opencti#8967]
## Description

In the internal demo environment.

Email notification is not correct.

1. Configure a very simple trigger:

![Image](https://github.com/user-attachments/assets/218953e8-94d7-4feb-ae13-c62f1f1e9071)

2. Change the label and see the history: 
![Image](https://github.com/user-attachments/assets/da7f63b6-230c-4898-a08b-54c1e99f5115)

3. Email notification:
![Image](https://github.com/user-attachments/assets/bf6aedc7-decf-466a-9810-2c917b20f5f7)

=> Not any insight in the email about what's going on...

**Comment by nino-filigran** [comment]
Ok so the fix is at least to show what's been updated/deleted/added in the entity.

**Comment by SarahBocognano** [comment]
Wasn't it fixed with the last notification buffering feature ? @nino-filigran

**Comment by nino-filigran** [comment]
Yes, I just tested and I do not reproduce anymore. The same trigger than above on our testing instance provides me the follwoing result: 

![Image](https://github.com/user-attachments/assets/e7423c15-5d71-44c4-8c7c-68aeec81b434)
